apps/api/prisma/seed/fields.ts

```diff
@@ -0,0 +1,31 @@
+import { PrismaClient } from '@prisma/client';
+const prisma = new PrismaClient();
+export async function SeedFields() {
```

Comment:
Ao invés de instanciar varios PrismaClient você poderia passar como parametro:

```suggestion
export async function SeedFields(prisma: PrismaClient) {
```